experimental/FTheoryTools/src/LiteratureTateModels/database.json

```diff
@@ -0,0 +1,23 @@
+[
+{
+    "arxiv_id": "1109.3454",
+    "equ_nr": "3.5",
+    "description": "SU(5)xU(1) restricted Tate model",
+    "dim": 3,
+    "variables": ["a10", "a21", "a32", "a43", "a65", "w"],
+    "a1": [[1, 1]],
+    "a2": [[2, 1], [6, 1]],
+    "a3": [[3, 1], [6, 2]],
+    "a4": [[4, 1], [6, 3]],
+    "a6": [],
+    "resolutions": [[["x", "y", "w"], ["y", "e1"], ["x", "e4"], ["y", "e2"], ["x", "y"], ["e1", "e4", "e2", "e3", "s"]],
+                    [["x", "y"], ["y", "s", "w"], ["s", "e4"], ["s", "e3"], ["s", "e1"], ["s", "w", "e3", "e1", "e2"]]]
+
+},
+{
+    "arxiv_id": "xxxxxxxx",
+    "equ_nr": "xxxxxx",
+    "description": "some other model",
+    "dim": 3
+}
+]
```

Comment:
This lacks almost all metadata necessary. Using the OSCAR serialization will already add a substantial amount.

Reply:
parts of the serialization will have to be extended, but anything mathematical should be using the serialization tools, and if they aren't there we should have a discussion on what is needed. Handling the meta data is also something we would like to implement

Reply:
Just to summarize my discussion last Friday with @lkastner and @antonydellavecchia - I fully agree.

This here, should (including with quite a few technical improvements as pointed out by @fingolfin) be a working version that we can improve.

@lkastner : As agreed last Friday, I will not make changes to this branch until I hear from you. From this point on, it is all yours. Please feel free to propose/push changes that marry this humble attempt with the existing serialization techniques in OSCAR.